lib/network.go

```diff
@@ -1890,6 +1890,11 @@ type MsgDeSoHeader struct {
 	// mined.
 	TstampSecs uint64
 
+	// TODO: Add a new TstampNanoSecs field that will have nanosecond resolution.
+	// For backwards compatibility with existing backends and frontends, we will keep
+	// the existing TstampSecs field and populate it as TstampNanoSecs / 1e9 when decoding
+	// block headers from bytes.
```

Comment:
I was thinking about this before reading this comment. I think this approach makes sense. Maybe we rename TstampSecs to LegacyTstampSecs.

Reply:
This code comment is resolved here: https://github.com/deso-protocol/core/pull/631
Marking this as resolved since you've reviewed PR 631 already and like the solution there.